### PR TITLE
Migration: Corporate Billing

### LINKED ISSUE #1061: Migrate article: Corporate billing
Migrate the content from this article to fit the documentation portal format https://help.aiven.io/en/articles/1786556

(Assign this issue to yourself before you start, and remember to mention the issue number when you open your pull request)

### PR DESCRIPTION
**Pull request by sluhur**
This is to migrate issue 1061 
Fix #1061

# What changed, and why it matters

### CODE REVIEW

File: docs/platform/concepts/corporate-billing.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+=================
+
+Receive invoices instead of credit card charges for your Aiven services
+
+
+The **default billing** method for all new Aiven customers is a credit card charge at the end of the month for all charges accrued over that month. Each Aiven project is charged separately.
+
+The billing unit for Aiven services is 1 hour. We automatically calculate the amount based on the services that have been running in your Aiven project. 
+
+For customers who prefer to pay by invoice or are unable to pay with a credit card, we also offer corporate billing options.  The standard corporate billing option is monthly invoices generated at the end of the month based on actual usage, similar to the credit card charge method used by default.
```
Comment (ftisiot) [inline docs/platform/concepts/corporate-billing.rst:11]:
```suggestion
For customers who prefer to pay by invoice or are unable to pay with a credit card, we also offer corporate billing options. The standard corporate billing option generates monthly invoices at the end of the month based on actual usage, similar to the credit card charge method used by default.
```

File: docs/platform/concepts/corporate-billing.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+These invoices are automatically generated by our backend and sent as PDFs over email to the billing email addresses configured in the Aiven dashboard.
```
Comment (ftisiot) [inline docs/platform/concepts/corporate-billing.rst:13]:
```suggestion
These invoices are automatically generated by our backend and sent as PDF over email to the billing email addresses configured in the Aiven dashboard.  
```

File: docs/platform/concepts/corporate-billing.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+-  Norwegian kroner
+
+-  Swedish kronor
+
+-  Australian dollars 
+
+-  Canadian dollars 
+
+Invoices must be paid within 10 days of receipt by a wire transfer to Aiven bank account at Citibank Europe Plc:
```
Comment (ftisiot) [inline docs/platform/concepts/corporate-billing.rst:35]: We might need to add some words to the vocabulary here

**Review by ftisiot — CHANGES_REQUESTED** [review]
Thanks @sluhur,
Left Some comments

**Comment by ftisiot** [comment]
@harshini-rangaswamy I know you're looking at billing topics, and I'm not sure if the shape of this article is what we're looking for, can you give suggestions?

**Comment by ftisiot** [comment]
Hi @sluhur if you fixed some of the comments can you please also mark the related conversation as resolved? This helps with assessing what has been fixed already (and applies to the other PRs :D )

File: docs/platform/concepts/corporate-billing.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+-  Norwegian kroner
+
+-  Swedish kronor
+
+-  Australian dollars 
+
+-  Canadian dollars 
+
+A payment due upon receipt is a payment that customers must make immediately upon receiving the invoice for a transaction. Invoice must be paid within **10 days** of receipt by a wire transfer to Aiven bank account at Citibank Europe Plc:
```
Comment (harshini-rangaswamy) [inline docs/platform/concepts/corporate-billing.rst:34]: This line is not very clear to me. Maybe rewrite it?

"A payment due upon receipt is a payment that customers must make immediately upon receiving the invoice for a transaction."

Comment (sluhur) [reply docs/platform/concepts/corporate-billing.rst]: I'm deleteting this line "A payment due upon receipt is a payment that customers must make immediately upon receiving the invoice for a transaction."

**Comment by ftisiot** [comment]
@harshini-rangaswamy can you please check this again?

File: docs/platform/concepts/corporate-billing.rst [hunk 5]
```diff
@@ -0,0 +1,49 @@
+Corporate billing
+=================
+
+Everyone has their preferred way of paying for services. At Aiven, we give you the flexibility to receive invoices or pay by credit card for your Aiven services.
+
+The **default billing** method for all new Aiven customers is a credit card charge at the end of the month for all charges accrued over that month. Each Aiven project is charged separately.
+
+The billing unit for Aiven services is by the clock hour. The charges are automatically calculated based on the services running in your Aiven project. 
+
+We offer corporate billing options for customers who prefer to pay by invoice. The standard corporate billing option is monthly invoices generated at the end of the month based on actual usage, similar to the credit card charge method used by default.
```
Comment (harshini-rangaswamy) [inline docs/platform/concepts/corporate-billing.rst:10]:
```suggestion
We offer corporate billing options for customers who prefer to pay by invoice. The standard corporate billing option is monthly invoices generated at the end of the month based on actual usage, similar to the credit card billing method.
```

**Review by harshini-rangaswamy — COMMENTED** [review]
One minor change. Otherwise, it looks good.

**Review by harshini-rangaswamy — APPROVED** [review]
Looks good, thanks!

**Comment by ftisiot** [comment]
I made some changes to the format to be more readable. @harshini-rangaswamy the last sentence

> To switch from credit card charges to corporate billing, please make sure your billing profile and email address are up to date on your Aiven project’s billing settings as shown below and contact [sales@Aiven.io](mailto:sales%40Aiven.io) to request the change.

Shouldn't this be part of a dedicated howto article instead of being in the concepts area?

**Comment by harshini-rangaswamy** [comment]
> I made some changes to the format to be more readable. @harshini-rangaswamy the last sentence
> 
> > To switch from credit card charges to corporate billing, please make sure your billing profile and email address are up to date on your Aiven project’s billing settings as shown below and contact [sales@Aiven.io](mailto:sales%40Aiven.io) to request the change.
> 
> Shouldn't this be part of a dedicated howto article instead of being in the concepts area?

Ideally, yes. Also, I think a screenshot is missing here as it says, " Aiven project’s billing settings as shown below". Maybe, for now, we could add the screenshot. The billing module will change soon, we would need to re-write all these topics again. (https://aiven.atlassian.net/browse/ROADMAP-171)

**Comment by ftisiot** [comment]
@sluhur can you please add the screenshot?
Thanks!

**Comment by sluhur** [comment]
> @sluhur can you please add the screenshot? Thanks!

Done

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**